### PR TITLE
Issue 155 - update docs for 4.0 - extract JSON models documentation

### LINKED ISSUE #155: Document new validation models with JSON schemas
* [x] Extract internal documentation to https://github.com/IGNF/validator-schema#ignfvalidator-schema (french)
* [ ] Check consistency with implementation (create 4.1 issues for missing features)
* [ ] Review "doc/" to describe implementation and avoid duplication

### PR DESCRIPTION
**Pull request by mborne**
Sylvain, could you have a look to the updated README and linked docs please : 

https://github.com/IGNF/validator/tree/issue_155#validator

Note that IGNF/validator-schema is pointed by this page : https://github.com/IGNF/validator/blob/issue_155/doc/model.md#mod%C3%A9lisation-des-donn%C3%A9es

(closes #155)

### CODE REVIEW

File: doc/model.md [hunk 1]
```diff
@@ -1,162 +1,31 @@
...
+Le schéma ci-après illustre les relations entre les modèles :
 
-Remarque : Le modèle de document est porteur d'une expression régulière qui permet de valider le nom du dossier à valider.
+![Diagramme de classe](img/class-diagram.jpeg)
 
-## Modèle de fichier (FileModel)
+## Format de table supportés
 
-Le modèle de fichier est identifié par au nom au sein du modèle de document. Le fichier est décrit par :
+On nottera que :
```
Comment (slafayIGN) [inline doc/model.md:21]: typo "on notera que"

File: doc/model.md [hunk 2]
```diff
@@ -1,162 +1,31 @@
...
-
-Remarque : La normalisation des données est en pratique le premier post-traitement.
+* Les formats suivants sont supportés :
+  * Shapefile
+  * Mapinfo
+  * CSV
+  * GML (Simple Feature, i.e. une seule table dans le GML)
+* Le champ géométrique doit être décrit sous le nom "WKT"
+
+Remarque : Ce dernier point est lié au comportement par défaut de ogr2ogr lors de la conversion en CSV. Une prochaine version du validateur apportera de la souplesse sur ce point.
```
Comment (slafayIGN) [inline doc/model.md:31]: Is there an open issue concerning the name of the geometry field ?
Maybe we can be specific on which target version (4.1 ?) will be more permissive.

File: doc/principe.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
+
+# Principe de fonctionnement du validateur de document
+
+## 1) Chargement du modèle de document
+
+Le modèle de document est chargé en fonction des options `--config/--version`.
```
Comment (slafayIGN) [inline doc/principe.md]: I think it lacks details. We should be more specific about what each of both options does.

File: doc/principe.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+## 2) Préparation d'un dossier de validation
+
+Le validateur prépare un répertoire de travail pour la validation où il écrira :
+
+* Le rapport de validation (`validation/validation.jsonl`)
```
Comment (slafayIGN) [inline doc/principe.md:12]: typo : one extra character in file extension

File: doc/model.md (hunk 2 above)
Comment (mborne) [reply doc/model.md]: Issue added with milestone 4.1 : https://github.com/IGNF/validator/issues/192

File: doc/principe.md (hunk 4 above)
Comment (mborne) [reply doc/principe.md:12]: No, jsonl stands for [JSON Lines](http://jsonlines.org/)

File: doc/principe.md (hunk 3 above)
Comment (mborne) [reply doc/principe.md]: `--model=<DOCUMENT_MODEL_URL>` describe to simplify (`--config/--version` is the legacy way)

**Pull request merged**